src/relay/qnn/op/op_common.h

```diff
@@ -286,7 +346,10 @@ static inline bool QnnBroadcastRel(const Array<Type>& types, int num_inputs, con
       .add_argument("rhs_zero_point", "Tensor", "The zero_point of the rhs tensor.")               \
       .add_argument("output_scale", "Tensor", "The scale of the output tensor.")                   \
       .add_argument("output_zero_point", "Tensor", "The zero_point of the output tensor.")         \
+      .add_argument("lhs_axis", "Tensor", "The channel quantization of the lhs tensor.")           \
+      .add_argument("rhs_axis", "Tensor", "The channel quantization of the rhs tensor.")           \
       .add_type_rel("QnnBroadcast", QnnBroadcastRel)                                               \
+      .set_attr<TOpPattern>("TOpPattern", kOpaque)                                                 \
```

Comment:
Yes. At some point I was getting an error since this pattern was missing, but now it's gone (tried a few different things in this branch). So, I'm gonna remove it.